### PR TITLE
animated spans explosions

### PR DESCRIPTION
**Pull request by CharlieMantium**

### CODE REVIEW

File: src/components/LandingAnimation/LandingAnimation.js [hunk 1]
```diff
@@ -88,15 +106,22 @@ const LandingAnimation = () => {
       >
         Press to say &apos;Hello!&apos;
       </StartButton>
-      <StyledSpan ref={spanHiRef}>Hi!</StyledSpan>
-      <StyledSpan ref={spanMyRef}>My</StyledSpan>
-      <StyledSpan ref={spanNameRef}>name</StyledSpan>
-      <StyledSpan ref={spanIsRef}>is</StyledSpan>
-      <StyledSpan ref={spanKarolRef}>Karol</StyledSpan>
+      <StyledSpan ref={hiSpanRef}>Hi!</StyledSpan>
```
Comment (Lukasz-Semik) [inline src/components/LandingAnimation/LandingAnimation.js:109]: Will it be translated?

File: src/components/SVG/Cloud.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+const CloudWrapper = styled.div`
+  display: flex;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -70%);
+  width: 100%;
+`;
+
+const Cloud = React.forwardRef(({ cloudSrc }, ref) => (
```
Comment (Lukasz-Semik) [inline src/components/SVG/Cloud.js]: I think that `src` would be totally finy

File: src/components/SVG/Cloud.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  <CloudWrapper>
+    <StyledImg ref={ref} src={cloudSrc} alt="Cloud of Dust" />
```
Comment (Lukasz-Semik) [inline src/components/SVG/Cloud.js]: `alt` should be translated

File: src/components/LandingAnimation/LandingAnimation.js (hunk 1 above)
Comment (CharlieMantium) [reply src/components/LandingAnimation/LandingAnimation.js:109]: yes, TODO comment placed and translation will be done in separate PR

**Review by Lukasz-Semik — APPROVED** [review]

**Pull request merged**